knowledge_graph/models.py

```diff
@@ -38,6 +38,22 @@ def load_user(id):
 
 
 class Scores(db.Model):
+    session_id = db.Column(
+        db.Float, db.ForeignKey("user.id"), primary_key=True, unique=True
```

Comment:
This also needs to get updated as an Integer rather than a float, correct?

Reply:
Yep, that makes sense- although maybe a string would be even better? I don't think a UUID can really be represented as an int, and casting it into one increases the chances of a collision, since  int is smaller- it should probably be a string. Would changing it to a string break anything that uses the db?